src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/attribute_editor.cljs

```diff
@@ -56,98 +56,66 @@
+     (create-section
+       (when (or (:saving? @state) (:deleting? @state))
+         [comps/Blocker {:banner "Updating..."}])
+       [:div {}
+        (map-indexed
+          (fn [i a]
+            [:div {}
+             [:div {:style {:float "left" :marginRight "0.5em"}}
+              (style/create-text-field
+                {:ref (str "field" i)
+                 :value (first a)
+                 :onChange #(swap! state update-in [:attrs-list i]
+                             assoc 0 (-> % .-target .-value))
+                 :disabled (or (not (:editing? @state))
+                             (contains? (:reserved-keys @state) i))
+                 :style (if (or (contains? (:reserved-keys @state) i)
+                              (not (:editing? @state)))
+                          {:backgroundColor (:background-gray style/colors)}
+                          {:backgroundColor "#fff"})})]
+             [:div {:style {:float "right"}}
+              (style/create-text-field
+                {:value (second a)
+                 :onChange #(swap! state update-in [:attrs-list i]
+                             assoc 1 (-> % .-target .-value))
+                 :disabled (not (:editing? @state))
+                 :style (if-not (:editing? @state)
+                          {:backgroundColor (:background-gray style/colors)}
+                          {:backgroundColor "#fff"})})
+              (icons/font-icon
+                {:style {:paddingLeft "0.5em" :padding "1em 0.7em"
+                         :color "red" :cursor "pointer"
+                         :display (when-not (:editing? @state) "none")}
+                 :onClick (fn [e]
+                            (if (contains? (:reserved-keys @state) i)
+                              (do
+                                (swap! state assoc :deleting? true)
+                                (delete-attribute props (first a)
+                                  (fn [e] (swap! state #(-> %
+                                                         (assoc :deleting? false)
+                                                         (update-in [:reserved-keys] utils/delete i)
+                                                         (update-in [:attrs-list] utils/delete i))))))
+                              (swap! state update-in [:attrs-list] utils/delete i)))}
+                :x)]
+             (common/clear-both)])
+          (:attrs-list @state))
+        [:div {:style {:display (when-not (:editing? @state) "none")}}
+         [comps/Button {:style :add :text "Add new"
+                        :onClick
+                        #(do
+                          (swap! state update-in [:attrs-list] conj ["" ""])
+                          (js/setTimeout (fn [] (common/focus-and-select
+                                           (-> (@refs (str "field" (- (count (:attrs-list @state)) 1))) .getDOMNode))) 250 )
+                          )
+                        }]]])
```

Comment:
Fix the formatting around here, and I think the timeout delay can be quite a bit shorter.